packages/node-firebird-driver/src/test/tests.ts

```diff
@@ -22,10 +57,12 @@ export function runCommonTests(client: Client) {
 		let tmpDir: string;
 
 		function getTempFile(name: string): string {
-			return `${tmpDir}/${name}`;
+			const p = `${(baseTmpDir || tmpDir)}/${name}`;
+			return `${host || '127.0.0.1'}:${p}`;
```

Comment:
Looks like you make it not possible to use embedded any more?

Reply:
So, what are the connection requirements to use the embedded server? Omit the host part?

Reply:
Yes.